portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,23 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Ihsan Olawale's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+      <h1>Ihsan Olawale's Portfolio</h1>
+      <p>This is Ihsan Olawale's portfolio.</p>
+      <p>I, Ihsan Olawale, am a STEP intern at Google. By the end of the internship I will have created a web project. This is practice.</p>
       <p>Click here to get a random message:</p>
       <button onclick="addRandomGreeting()">Hello</button>
       <div id="greeting-container"></div>
+      <p>Here's what I look like, at least when looking at my Google profile:</p>
+      <img src="/images/me.png" alt="What I look like" width=300 height=300>
```

Comment:
nit - let's use [double quotes](https://google.github.io/styleguide/htmlcssguide.html#HTML_Quotation_Marks) for attributes. it's XML standard.

Reply:
Okay, so far it looks like I've conformed, given that I didn't use single quotes for the attributes.

Reply:
Ah - I should have been more clear - we should go ahead and add double quotes around the `width` and `height` attributes on this `img` element.

Reply:
Oh I see, I will do that then.